Formula/unum.rb

```diff
@@ -0,0 +1,16 @@
+class Unum < Formula
+  desc "Interconvert numbers, Unicode, and HTML/XHTML entities"
+  homepage "https://www.fourmilab.ch/webtools/unum/"
+  url "https://www.fourmilab.ch/webtools/unum/prior-releases/3.2/unum.tar.gz"
```

Comment:
Why does this say `prior-releases`? Is it not the last version?

Reply:
It looks like that directory is just an index of all releases, while the main download link on the front page points to an unversioned tarball.